Makefile

```
@@ -58,7 +58,7 @@ test:
 ## unit : Runs unit tests without coverage and race detection.
 .PHONY: unit
 unit:
-	@$(foreach var,$(PROJECTS),$(MAKE) -C $(var) unit || exit 1;)
+	@$(foreach var,$(PROJECTS), cd $(CURDIR)/$(var) && mage unitTest || exit 1;)
```

Btw, instead of changing directory I think you can use the `-d` flag.
```suggestion
	@$(foreach var,$(PROJECTS), mage -d $(CURDIR)/$(var) unitTest || exit 1;)
```